iree/compiler/Dialect/HAL/Target/VMLA/BUILD

```diff
@@ -12,12 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
+
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+iree_cmake_extra_content(
+    content = """
+if(NOT ${IREE_TARGET_BACKEND_VMLA})
+  return()
+endif()
+""",
+)
```

Comment:
Maybe have bazel_to_cmake call dedent on this, rather than having to make the indent all wonky?

Reply:
`dedent` is pretty simple, but I'd still rather not add extra processing.